### DISCUSSION TITLE
Making funding projects easier

### DISCUSSION THREAD

**Discussion by TheJoeFin** [microsoft/winget-cli#3627] [Ideas]
How could WinGet make a positive impact on the challenge of funding free software? Installing all the software I need for my device with a simple script and no clicking is amazing, but frequently the appeal to support a project gets side stepped. In an effort to make the Windows software community more vibrant and sustainable could WinGet build a `fund` command similar to NPM?

https://docs.npmjs.com/cli/v6/commands/npm-fund

Curious to hear other ideas people have on the topic of funding free software and open source in general.

**Comment by denelon** [comment]
We have a feature that was submitted to help support FOSS:
* https://github.com/microsoft/winget-cli/issues/3398
